openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala

```diff
@@ -313,39 +322,45 @@ object OpenApi {
 
 final class Info private (
     val title: String,
-    val version: String
+    val version: String,
+    val description: Option[String]
 ) extends Serializable {
 
   override def toString: String =
-    s"Info($title, $version)"
+    s"Info($title, $version, $description)"
 
   override def equals(other: Any): Boolean =
     other match {
-      case that: Info => title == that.title && version == that.version
-      case _          => false
+      case that: Info =>
+        title == that.title && version == that.version && description == that.description
+      case _ => false
     }
 
   override def hashCode(): Int =
     Hashing.hash(title, version)
```

Comment:
You should add `description` here as well.

Reply:
Missed, I'll update.